Tools/build/generate_stdlib_module_names.py

```diff
@@ -70,8 +70,16 @@ def list_packages(names):
         package_path = os.path.join(STDLIB_PATH, name)
         if not os.path.isdir(package_path):
             continue
-        if any(package_file.endswith(".py")
-               for package_file in os.listdir(package_path)):
+        # Walk the package directory to check if the package is a namespace
```

Comment:
I'd avoid using the term "namespace" here, it already led to a lot of confusion in the PEP discussion
```suggestion
        # Walk the package directory to check if there are nested packages
```

Reply:
I think in this case "namespace" is required as the reason `compression` isn't caught by default is it is a namespace package. `compression` is currently a namespace package, but we wanted to be clear it didn't need to be that way in the future in the PEP (hence the confusion).

Nested packages doesn't clarify that this is solving the case where the package is a PEP 420 namespace package in terms of layout. I guess we could say "nested packages with no modules under the top-level package"?

Reply:
is there any reason **not** to add the empty `Lib/compression/__init__.py` file so it gets treated as a normal package instead of a PEP 420 style namespaces package?  We expect no code to be in the top level package and for users to always be using `from compression import thing`.

I don't _believe_ we have any namespaces packages in the stdlib today (in my quick check).

Reply:
There aren't, and there's no reason we can't add `Lib/compression/__init__.py`. My only concern is if someone does `import compression` and doesn't get anything they'll be confused. I think it is fine to to add it however.

Reply:
I've added the `__init__.py` file and reverted the changes against `generate_stdlib_module_names.py`.